python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/aaa_accounting.schema.yml

```diff
@@ -22,8 +22,29 @@ keys:
               group:
                 description: Group Name.
                 type: str
+                deprecation:
+                  warning: true
+                  new_key: method[].group
+                  remove_in_version: v6.0.0
               logging:
                 type: bool
+                deprecation:
+                  warning: true
+                  new_key: method[].method
+                  remove_in_version: v6.0.0
+              methods:
```

Comment:
add `min_length` of 1 so that you know it will never be empty and it should simplify your if statements in the templates